mkdocs/docs/HPC/interactive_debug.md

```diff
@@ -1,21 +1,23 @@
-# Interactive and debug cluster { #ch:interactive_ugent}
+# Interactive and debug cluster
 
-## Purpose { #sec:interactive_ugent_pupose}
+## Purpose
 
 The purpose of this cluster is to give the user an environment where
 there should be no waiting in the queue to get access to a limited
 number of resources. This environment allows a user to immediatelty
 start working, and is the ideal place for interactive work such as
 development, debugging and light production workloads (typically
 sufficient for training and/or courses).
-<br></br>This enviroment should be seen as an extension of the login nodes,
+<br>
```

Comment:
The `<br></br>` was deliberate, to start a new paragraph.

It's now one bigger block of text, which is harder to parse imho...